test/cli-validator/tests/optionHandling.js

```diff
@@ -97,6 +97,33 @@ describe('cli tool - test option handling', function() {
     expect(validatorsPrinted).toEqual(true);
   });
 
+  it('should print only errors when the -e command is given', async function() {
+    const capturedText = [];
+
+    const unhookIntercept = intercept(function(txt) {
+      capturedText.push(stripAnsiFrom(txt));
+      return '';
+    });
+
+    const program = {};
+    program.args = ['./test/cli-validator/mockFiles/errAndWarn.yaml'];
+    program.only_errors = true;
+    program.default_mode = true;
+
+    await commandLineValidator(program);
+    unhookIntercept();
+
+    let errorsOnly = false;
+
+    capturedText.forEach(function(line) {
+      if (line.includes('error')) {
+        errorsOnly = true;
+      }
+    });
```

Comment:
This test doesn't do what you want it to do. If there are both errors and warnings in the output, this will still pass. Will you rethink this test?

Reply:
This test is still not totally correct, will you address it?